### PR TITLE
Add twitter to replace news coverage

### PR DESCRIPTION
**Pull request by gonuke**
Hopefully this will facilitate keeping things more up-to-date since it won't require a PR to add news items!

### CODE REVIEW

File: source/archive.rst [hunk 1]
```diff
@@ -1,6 +1,26 @@
 News Archive
 -------------
 
+* **October 2016** Watch `Meghan McGarry <people/mcgarry.html>`_ give a talk on using |Cyclus| for  `State-Level Decision-Making in a Regional Model of Proliferation Risk <https://youtu.be/3HySc2oVYQU?list=PLmNrnoyQ30KvxZdAAz2MPDv-1lkTcUS>`_ at the `2016 CVT Workshop <https://cvt.engin.umich.edu/cvt-workshop-2016/>`_ in Ann Arbor.
+  
+* **October 2016** CNERG begins support of the `Spallation Neutron
+  Source <https://neutrons.ornl.gov/sns>`_ and its design of a `second target
+  station <https://neutrons.ornl.gov/sts>`_ using DAG-MCNP6.
+
+* **August, 2016** `Robert Carlsen <people/calrsen.html>`_ is the lead author
```
Comment (bam241) [inline source/archive.rst]: dead link to Robert pages

File: source/archive.rst [hunk 2]
```diff
@@ -1,6 +1,26 @@
...
+  on a new paper `Challenging Fuel Cycle Modeling Assumptions: Facility and
+  Time-Step Discretization Effects
+  <http://www.ans.org/pubs/journals/nt/a_38930>`_ in Nuclear Technology.
+
+* **July 21, 2016** `Elliott Biondo <people/biondo.html>`_ defended his PhD
```
Comment (bam241) [inline source/archive.rst]: same here for Elliott

File: source/archive.rst [hunk 3]
```diff
@@ -1,6 +1,26 @@
...
+  dissertation, describing a complete approach to variance reduction for
+  shutdown dose rate problems and demonstrating substantial speedup in such
+  problems.
+
+* **March 29, 2016** `Robert Carlsen <people/carlsen.html>`_ defended his PhD
```
Comment (bam241) [inline source/archive.rst]: same

File: source/archive.rst [hunk 4]
```diff
@@ -1,6 +1,26 @@
...
+  dissertation, highlighting contributions in nuclear fuel cycle optimization
+  and the concept of hedging strategies for nuclear technology deployment.
+
 * **March, 2016** `Elliott Biondo <people/biondo.html>`_ is the lead author on
```
Comment (bam241) [inline source/archive.rst]: same

**Comment by bam241** [comment]
thank you @gonuke

**Comment by gonuke** [comment]
Are you pushing an update?

**Comment by bam241** [comment]
I did

**Pull request merged**